CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ feature is a good idea for hub if it improves some workflow for a GitHub user.
 
 ## How to install dependencies and run tests
 
-1. [Clone the project](./README.md#source)
+1. [Clone the project](./README.md#source) by using fork in the current project
```

Comment:
Sorry, I don't understand what this is about. The current instructions are clear (it's okay to clone `github/hub`; one doesn't need to clone a fork) and you haven't explained why the updated text is an improvement. Closing